src/shapes/itext.class.js

```diff
@@ -462,23 +488,26 @@
      * @param {Object} boundaries Object with left/top/leftOffset/topOffset
      * @param {CanvasRenderingContext2D} ctx transformed context to draw on
      */
-    renderSelection: function (boundaries, ctx) {
+    renderSelection: function (ctx, boundaries) {
```

Comment:
i moved those argument orders, in general, because we put ctx as first argument when we render.